iceoryx_posh/source/roudi/port_pool.cpp

```diff
@@ -90,7 +90,7 @@ PortPool::addRunnableData(const ProcessName_t& process,
 }
 
 cxx::expected<popo::ConditionVariableData*, PortPoolError>
-PortPool::addConditionVariableData(const ProcessName_t& process)
+PortPool::addConditionVariableData(const ProcessName_t& process[[gnu::unused]])
```

Comment:
@mossmaurice, was this intentional to have a unused variable?

Reply:
I think we don't need the `const ProcessName_t&` here. Could you please remove it?

Reply:
done